### PR TITLE
Feature/v haiboz

### PR DESCRIPTION
**Pull request by v-haiboz**
#BUG 232189:<seealso> links are dropped.

### CODE REVIEW

File: test/yml_SDP/CatLibrary.Cat-2.yml [hunk 1]
```diff
@@ -186,6 +189,10 @@ examples: >-
   </code></pre>
 
   <p>As you see, here we bring in <strong>pointer</strong> so we need to add <code data-dev-comment-type="languageKeyword">unsafe</code> keyword.</p>
+source:
```
Comment (TianqiZhang) [inline test/yml_SDP/CatLibrary.Cat-2.yml]: this shouldn't be here

Comment (v-haiboz) [reply test/yml_SDP/CatLibrary.Cat-2.yml]: sure

File: ECMA2Yaml/ECMAHelper/ECMALoader.docs.cs [hunk 2]
```diff
@@ -119,6 +119,18 @@ public Docs LoadDocs(XElement dElement,string filePath)
...
 
+        /// <summary>
+        /// merge altmember and seealso to seeAlso of yml file.
+        /// </summary>
+        /// <param name="dElement">a Xlement</param>
+        /// <returns>List<string></returns>
+        private static List<string> MergeAltmemberAndSeealsoToAltMemberCommentsIds(XElement dElement)
+        {
+            var ids = dElement.Elements("altmember")?.Select(alt => alt.Attribute("cref").Value).ToList();
+            ids.AddRange(dElement.Elements("seealso")?.Select(alt => alt.Attribute("cref").Value).ToList());
```
Comment (TianqiZhang) [inline ECMA2Yaml/ECMAHelper/ECMALoader.docs.cs]:
```suggestion
            ids.ConcatList(dElement.Elements("seealso")?.Select(alt => alt.Attribute("cref").Value).ToList());
```

Comment (v-haiboz) [reply ECMA2Yaml/ECMAHelper/ECMALoader.docs.cs]: fixed

**Pull request closed without merging**